packages/flutter_tools/test/general.shard/fuchsia/fuchsia_device_test.dart

```diff
@@ -533,19 +533,20 @@ void main() {
         throwsA(equals('Failed to copy screenshot from device:\n<error-message>')),
       );
     }, overrides: <Type, Generator>{
-      ProcessManager: () => mockProcessManager,
+      ProcessManager: () => MockProcessManager(),
+      FileSystem: () => MemoryFileSystem.test(),
```

Comment:
I'm not sure how any of these passed with a missing file system override, but it looked to be using the real file system sometimes